style.css

```diff
@@ -1,575 +1,829 @@
+  display: none;
 }
 
-.hex {
-  color: #6070ff;
+section.visible {
+  display: block;
+}
+
+div.menu-link {
+  display: flex;
+  flex-direction: column;
 }
 
-a:hover {
+section.mobile-menu a {
+  margin-bottom: 30px;
   text-decoration: none;
-  cursor: pointer;
+  color: white;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
 }
 
-.main-wrapper {
-  background: white;
-  display: flex;
-  flex-direction: column;
-  border-bottom-left-radius: 80px;
+.logo {
+  font-family: "Poppins", sans-serif;
+  color: #6070ff;
+  font-size: 1.8rem;
+  font-weight: 700;
+  line-height: 2rem;
+  text-decoration: none;
 }
 
-.open body {
-  background-color: #6070ff;
+.links {
+  display: none;
 }
 
-.navbar {
-  width: 90%;
-  margin: auto;
-  padding-top: 1rem;
+.headline {
+  background-image: url("images/header-shapes\ mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  height: calc(718px - 45px);
+  padding-top: 3.7rem;
   background-color: #fff;
+  border-bottom: 2px solid #e9e9eb;
+  border-bottom-left-radius: 8rem;
 }
 
-.navbar .container-fluid .navbar-brand {
-  font-style: normal;
+.introduction {
+  font-family: "Poppins", sans-serif;
+  margin: 0 2.4rem 0 2.4rem;
+}
+
+.heading {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 4rem;
+  line-height: 5.2rem;
+  padding-top: 8.1rem;
+}
+
+.heading-desk {
+  display: none;
+}
+
+.intro-info {
+  font-size: 1.6rem;
+  font-weight: 400;
+  line-height: 2.4rem;
+  color: #344563;
+  margin-top: 1.2rem;
+}
+
+.connect {
+  color: #7f8cff;
   font-weight: 500;
-  font-size: 20px;
-  font-family: 'Poppins', sans-serif;
-  line-height: 20px;
+  font-size: 1.6rem;
+  line-height: 2.4rem;
 }
 
-.toolbar {
-  flex: 1 1 auto;
+.social-icons {
   display: flex;
-  height: 48px;
-  justify-content: space-between;
   align-items: center;
-  padding-bottom: 40px;
+  padding-left: 0;
+  gap: 2rem;
 }
 
-.logo-place {
-  margin-left: 12px;
+.list {
+  text-decoration: none;
+  list-style: none;
+  align-items: center;
 }
 
-.my-logo {
-  padding: 10px 12px 10px 12px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 18px;
-  line-height: 20px;
-  text-decoration: none;
+.works {
+  background: #f7f3f3;
+  padding: 11.4rem 0 0 0;
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+/* popup window   */
+
+.popup-window {
+  position: fixed;
+  z-index: 4;
+  display: none;
+  background: rgba(128, 128, 128, 0.5);
+  opacity: 1;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  backdrop-filter: blur(4px);
+  padding-bottom: 50px;
 }
 
-.headline {
-  /* width: 100%; */
-  background-image: url("images/header-shapes.png");
-  background-size: 100% 550px;
-  border-radius: 0 0 0 100px;
-  display: flex;
-  flex-direction: column;
+.modal-mobile {
+  position: relative;
+  top: 0;
+  left: 0;
+  margin-top: 18px;
+  margin-left: 16px;
+  margin-right: 16px;
```

Comment:
- [ ] Also for a better user experience, please make sure on mobile, the user can see the all content on the popup modal.
`You may consider adding a scroll on the popup`

![image](https://user-images.githubusercontent.com/97100091/225695887-4b2a8c57-9efe-4255-bbe8-de7cef10a82c.png)